src/System.Private.CoreLib/src/System/Enum.cs

```diff
@@ -269,7 +269,7 @@ private static string InternalFlagsFormat(RuntimeType eT, TypeValuesAndNames ent
             const int SeparatorStringLength = 2; // ", "
             string result = string.FastAllocateString(checked(resultLength + (SeparatorStringLength * (foundItemsCount - 1))));
 
-            Span<char> resultSpan = MemoryMarshal.CreateSpan(ref result.GetRawStringData(), result.Length);
+            Span<char> resultSpan = new Span<char>(ref result.GetRawStringData(), result.Length);
```

Comment:
nit: use var here

Reply:
Thanks.  It's pre-existing, I'm not going to reset CI for it.